solutions/web3-fetching-strategies/pages/index.tsx

```diff
@@ -0,0 +1,132 @@
+import Head from 'next/head'
+import { Layout, Text, Page, Link } from '@vercel/examples-ui'
+import { BAYCAbi__factory } from '../types/ethers-contracts'
+import { ethers } from 'ethers'
+import useSWR from 'swr'
+import { FC } from 'react'
+
+// import abi from '../lib/BAYC.abi.json'
+
+// const contractAddress = '0xBC4CA0EdA7647A8aB7C2061c2E118A18a936f13D'
+
+// const contract = new ethers.Contract(contractAddress, abi)
```

Comment:
Let's not keep context without any context, either remove them or mention why are they there.

Reply:
Removed!